docs/can-guides/topics/debugging.md

```diff
@@ -331,3 +331,99 @@ Component.extend({
     }
 });
 ```
+
+## Installing CanJS Devtools
+
+There is a CanJS Devtools Chrome Extension available in the [Chrome Web Store](https://chrome.google.com/webstore/detail/canjs-devtools/hhdfadlgplkpapjfehnjhcebebgmibcb).
+
+Once installed, you will see a CanJS icon next to the address bar:
+
+<img src="../../docs/can-guides/images/devtools/canjs-button-disabled.png"
```

Comment:
Are these image paths right? Could they be `../images/devtools/`?

Reply:
The images dont get copied from `docs` to `doc`, which is where the generated html is output. I think these are right. This is how other guides do paths for images.